components/server/src/dump.rs

```diff
@@ -223,6 +278,50 @@ mod tests {
         assert(15, &raft_engine);
     }
 
+    #[test]
+    fn test_dump_with_separate_wal_without_config_when_dump() {
```

Comment:
You can merge these two tests, doing two `check_and_dump_raft_db` in one run.

Reply:
Well I review here. I found it is not possible to merge theses tests. 'check_and_dump_raft_db' will rename the origin raftdb and delete it safely. To merge these test needs to generate the raftdb twice, too. So I think it's better to leave them alone here.